proxy/common.go

```diff
@@ -25,42 +23,6 @@ var (
 	Log = common.Log
 )
 
-func callWeave(args ...string) ([]byte, []byte, error) {
```

Comment:
We were calling `weave attach`, and that remains a valid command that users can call separately, so I don't think so.  It's plausible that we could trim down the argument-parsing it used to do - e.g. `--add-host` was not a documented option - but that's pretty low-benefit.